skyportal/app_server.py

```diff
@@ -143,35 +142,40 @@ def log_request(self, handler):
     (r'/api/sources(/[0-9A-Za-z-_\.\+]+)/classifications', ObjClassificationHandler),
     (r'/api/sources(/[0-9A-Za-z-_\.\+]+)/groups', ObjGroupsHandler),
     (r'/api/sources(/[0-9A-Za-z-_\.\+]+)/color_mag', ObjColorMagHandler),
-    (r'/api/(sources|spectrum)/([0-9A-Za-z-_\.\+]+)/comment', CommentHandler),
-    (r'/api/(sources|spectrum)/([0-9A-Za-z-_\.\+]+)/comment(/[0-9]+)?', CommentHandler),
+    (r'/api/(sources|spectra)/([0-9A-Za-z-_\.\+]+)/comments', CommentHandler),
+    (r'/api/(sources|spectra)/([0-9A-Za-z-_\.\+]+)/comments(/[0-9]+)?', CommentHandler),
     (
-        r'/api/(sources|spectrum)(/[0-9A-Za-z-_\.\+]+)/comment(/[0-9]+)/attachment',
+        r'/api/(sources|spectra)(/[0-9A-Za-z-_\.\+]+)/comments(/[0-9]+)/attachment',
         CommentAttachmentHandler,
     ),
     # Allow the '.pdf' suffix for the attachment route, as the
     # react-file-previewer package expects URLs ending with '.pdf' to
     # load PDF files.
     (
-        r'/api/(sources|spectrum)/([0-9A-Za-z-_\.\+]+)/comment(/[0-9]+)/attachment.pdf',
+        r'/api/(sources|spectra)/([0-9A-Za-z-_\.\+]+)/comments(/[0-9]+)/attachment.pdf',
         CommentAttachmentHandler,
     ),
     (
-        r'/api/(sources|spectrum)(/[0-9A-Za-z-_\.\+]+)/annotation',
+        r'/api/(sources|spectra)(/[0-9A-Za-z-_\.\+]+)/annotations',
         AnnotationHandler,
     ),
     (
-        r'/api/(sources|spectrum)(/[0-9A-Za-z-_\.\+]+)/annotation(/[0-9]+)?',
+        r'/api/(sources|spectra)(/[0-9A-Za-z-_\.\+]+)/annotations(/[0-9]+)?',
         AnnotationHandler,
     ),
-    (r'/api/sources(/[0-9A-Za-z-_\.\+]+)/annotations', ObjAnnotationHandler),
     (r'/api/sources(/.*)?', SourceHandler),
     (r'/api/source_notifications', SourceNotificationHandler),
     (r'/api/source_groups(/.*)?', SourceGroupsHandler),
+    (r'/api/spectra(/[0-9]+)?', SpectrumHandler),
+    (r'/api/spectra/parse/ascii', SpectrumASCIIFileParser),
+    (r'/api/spectra/ascii(/[0-9]+)?', SpectrumASCIIFileHandler),
+    (r'/api/spectra/range(/.*)?', SpectrumRangeHandler),
+    # DEPRECATED: To be removed in an upcoming release
```

Comment:
Maybe
```suggestion
    # FIXME: Deprecated, to be removed in an upcoming release
```
? PyCharm highlights these nicely, and it's easier to do grepping. We may also want to post a DNM (for now) PR where this is removed.

Reply:
Sure, will update 👍🏼